engine/schema/src/main/java/com/cloud/vm/dao/VMInstanceDaoImpl.java

```diff
@@ -325,6 +326,10 @@ protected void init() {
         VmsNotInClusterUsingPool.join("hostSearch2", hostSearch2, hostSearch2.entity().getId(), VmsNotInClusterUsingPool.entity().getHostId(), JoinType.INNER);
         VmsNotInClusterUsingPool.and("vmStates", VmsNotInClusterUsingPool.entity().getState(), Op.IN);
         VmsNotInClusterUsingPool.done();
+
+        ServiceOfferingSearch = createSearchBuilder();
+        ServiceOfferingSearch.and("serviceOfferingId", ServiceOfferingSearch.entity().getServiceOfferingId(), Op.EQ);
+        BackupSearch.done();
```

Comment:
Shouldn't it be ServiceOfferingSearch.done here?

Reply:
You're right